org.lflang/build.gradle

```diff
@@ -78,6 +78,7 @@ configurations {
 task getSubmoduleVersions(type: Exec) {
     description('Run a Git command to get the current status of submodules')
     workingDir project.rootDir
+    outputs.upToDateWhen { false }
```

Comment:
This will make gradle execute `git submodule status` every time updateRustRuntime is called.

Reply:
```suggestion
   // This will make gradle execute git submodule status every time updateRustRuntime is called
    outputs.upToDateWhen { false }
```